### ISSUE TITLE
Exception 2

### ISSUE THREAD

**Issue by desliem** [bluemurder/esp8266-ping#1]
Hi

I am getting a fatal exception (2) after a successful ping:

```
WiFi connected
IP address: 192.168.1.134
Attempt #1. Pinging 192.168.1.22
Echo response received from 192.168.1.22 in 212ms.

Exception (2):
epc1=0x3ffee1f0 epc2=0x00000000 epc3=0x00000000 excvaddr=0x3ffee1f0 depc=0x00000000

ctx: sys 
sp: 3ffffd40 end: 3fffffb0 offset: 01a0
```
This is the output of the ESP Exception Decoder in the Arduino IDE
```
Decoding 14 results
0x40227442: ping_coarse_tmr at /local/users/gauchard/arduino/arduino_esp8266/origin/tools/sdk/lwip2/builder/glue-lwip/esp-ping.c line 254
0x4022746d: ping_coarse_tmr at /local/users/gauchard/arduino/arduino_esp8266/origin/tools/sdk/lwip2/builder/glue-lwip/esp-ping.c line 263
0x40100690: free at /home/michel/Development/arduino-1.8.5/portable/packages/esp8266/hardware/esp8266/2.4.1/cores/esp8266/umm_malloc/umm_malloc.c line 1739
0x40210388: mem_free at /local/users/gauchard/arduino/arduino_esp8266/origin/tools/sdk/lwip2/builder/lwip2-src/src/core/mem.c line 152
0x4020a35c: cyclic_timer at /local/users/gauchard/arduino/arduino_esp8266/origin/tools/sdk/lwip2/builder/lwip2-src/src/core/timeouts.c line 165
0x402273b0: ping_coarse_tmr at /local/users/gauchard/arduino/arduino_esp8266/origin/tools/sdk/lwip2/builder/glue-lwip/esp-ping.c line 230
0x4020a428: sys_check_timeouts at /local/users/gauchard/arduino/arduino_esp8266/origin/tools/sdk/lwip2/builder/lwip2-src/src/core/timeouts.c line 351
0x40228bc0: ets_timer_handler_isr at ?? line ?
0x40228bcd: ets_timer_handler_isr at ?? line ?
0x40228c12: ets_timer_handler_isr at ?? line ?
0x40228813: ets_snprintf at ?? line ?
```
According to [https://gist.github.com/solusipse/b3a13e6578d97db97c27](url) an exception 2 is a 

> InstructionFetchErrorCause

 error.

I can run another ping library, [https://github.com/dancol90/ESP8266Ping](url), without problem on the same hardware: Wemos D1 mini without any connected pins.

Do you have any idea what I am doing wrong?

Regards.

**Comment by bluemurder** [comment]
Thank you for your report. I'll fix it as soon as possible.

**Comment by bluemurder** [comment]
Please try with last available 1.1.0 version, thank you

**Comment by desliem** [comment]
Version 1.1.0 now works correctly with the Wemos D1 mini.

I removed the example sketch in my first comment to avoid confusion. It would no longer work with version 1.1.0 of your library. I think your new examples will prove better for newbies.

Thank you for the fast turn around.
